notification-portlet-webapp/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,60 @@
+    except in compliance with the License. You may obtain a
+    copy of the License at:
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on
+    an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+<persistence version="2.0"
+  xmlns="http://java.sun.com/xml/ns/persistence" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://java.sun.com/xml/ns/persistence 
+  http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd">
+
+    <persistence-unit name="NoticePU" transaction-type="RESOURCE_LOCAL">
+        <provider>org.hibernate.ejb.HibernatePersistence</provider>
+        <class>org.jasig.portlet.notice.service.jpa.JpaAction</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaAddressee</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaAttribute</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaEntry</class>
+        <class>org.jasig.portlet.notice.service.jpa.JpaEvent</class>
+        <properties>
+            <!-- property name="hibernate.ejb.cfgfile" value="/notification.cfg.xml" /-->
+            <property name="hibernate.session_factory_name_is_jndi" value="false" />
+            <property name="hibernate.format_sql" value="true" />
+            <property name="hibernate.generate_statistics" value="true" />
+            <property name="hibernate.max_fetch_depth" value="3" />
+            <property name="hibernate.default_batch_fetch_size" value="4" />
+            <property name="hibernate.jdbc.batch_size" value="25" />
+            <property name="hibernate.id.new_generator_mappings" value="true" />
+
+            <property name="hibernate.cache.use_query_cache" value="true" />
+            <property name="hibernate.cache.use_second_level_cache" value="true" />
+            <!-- property name="hibernate.cache.provider_class" value=">net.sf.ehcache.hibernate.SingletonEhCacheProvider" /-->
+            <property name="hibernate.cache.region.factory_class" value="org.hibernate.cache.ehcache.SingletonEhCacheRegionFactory" />
+            <property name="hibernate.cache.use_minimal_puts" value="true" />
+
+            <property name="hibernate.current_session_context_class" value="thread" />
+            <!-- Ideally we'd prefer this off in production, but since the porlet (without uPortal overlay) uses in-memory
+                 hsqldb we need this to be update. Users *should* do a schema update before switching to new versions, or
+                 this should hopefully not do something negative if it ever does update the schema (typically doesn't). -->
+            <property name="hibernate.hbm2ddl.auto" value="update" />
+        </properties>
```

Comment:
Need to include an additional property: 
`<property name="hibernate.dialect" value="${environment.build.hibernate.dialect}"/>`
Postgres will not work without this being set.